resources/RELEASE_2022_04_08.md

```diff
@@ -0,0 +1,9 @@
+The week's update arrived and now the music module is now gone!
+_ _
+We hope our previous notice reached as many people as possible, and if you haven't, please read the previous announcement message for more information.
+_ _
+On other news, we have published Skyra to **App Discovery**, you'll soon be able to find or share her without a third-party website!
```

Comment:
```suggestion
In other news, we have published Skyra to **App Discovery**, you'll soon be able to find or share her without a third-party website!
```

You should really get Grammarly 
![image](https://user-images.githubusercontent.com/4019718/161437835-39170401-dd1c-4136-8ec9-b4912e08757e.png)